components/kubeflow/kfserving/src/kfservingdeployer.py

```diff
@@ -34,103 +34,231 @@
-    elif framework == 'sklearn':
-        return V1alpha2EndpointSpec(predictor=V1alpha2PredictorSpec(sklearn=V1alpha2SKLearnSpec(storage_uri=storage_uri)))
-    elif framework == 'xgboost':
-        return V1alpha2EndpointSpec(predictor=V1alpha2PredictorSpec(xgboost=V1alpha2XGBoostSpec(storage_uri=storage_uri)))
-    elif framework == 'onnx':
-        return V1alpha2EndpointSpec(predictor=V1alpha2PredictorSpec(onnx=V1alpha2ONNXSpec(storage_uri=storage_uri)))
-    elif framework == 'tensorrt':
-        return V1alpha2EndpointSpec(predictor=V1alpha2PredictorSpec(tensorrt=V1alpha2TensorRTSpec(storage_uri=storage_uri)))
+    if framework == "tensorflow":
+        return V1alpha2EndpointSpec(
+            predictor=V1alpha2PredictorSpec(
+                tensorflow=V1alpha2TensorflowSpec(storage_uri=storage_uri),
+                service_account_name=service_account,
+            )
+        )
+    elif framework == "pytorch":
+        return V1alpha2EndpointSpec(
+            predictor=V1alpha2PredictorSpec(
+                pytorch=V1alpha2PyTorchSpec(storage_uri=storage_uri),
+                service_account_name=service_account,
+            )
+        )
+    elif framework == "sklearn":
+        return V1alpha2EndpointSpec(
+            predictor=V1alpha2PredictorSpec(
+                sklearn=V1alpha2SKLearnSpec(storage_uri=storage_uri),
+                service_account_name=service_account,
+            )
+        )
+    elif framework == "xgboost":
+        return V1alpha2EndpointSpec(
+            predictor=V1alpha2PredictorSpec(
+                xgboost=V1alpha2XGBoostSpec(storage_uri=storage_uri),
+                service_account_name=service_account,
+            )
+        )
+    elif framework == "onnx":
+        return V1alpha2EndpointSpec(
+            predictor=V1alpha2PredictorSpec(
+                onnx=V1alpha2ONNXSpec(storage_uri=storage_uri),
+                service_account_name=service_account,
+            )
+        )
+    elif framework == "tensorrt":
+        return V1alpha2EndpointSpec(
+            predictor=V1alpha2PredictorSpec(
+                tensorrt=V1alpha2TensorRTSpec(storage_uri=storage_uri),
+                service_account_name=service_account,
+            )
+        )
     else:
-        raise("Error: No matching framework: " + framework)
+        raise ("Error: No matching framework: " + framework)
 
 
 def customEndpointSpec(custom_model_spec):
-    env = [client.V1EnvVar(name=i['name'], value=i['value']) for i in custom_model_spec['env']] if custom_model_spec.get('env', '') else None
-    ports = [client.V1ContainerPort(container_port=int(custom_model_spec.get('port', '')))] if custom_model_spec.get('port', '') else None
+    env = (
+        [
+            client.V1EnvVar(name=i["name"], value=i["value"])
+            for i in custom_model_spec["env"]
+        ]
+        if custom_model_spec.get("env", "")
```

Comment:
I don't think this is a valid syntax in python, you probably need to use backslash when breaking a new line.